File: src/backend/src/main/java/com/wissen/SmartInterviewProcess/models/Candidate.java
package com.wissen.SmartInterviewProcess.models;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Candidate {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private long id;

	private String name;

	private String phoneNum;

	private String fileName;

	public long getId() {
		return id;
	}

	public void setId(long id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getPhoneNum() {
		return phoneNum;
	}

	public void setPhoneNum(String phoneNum) {
		this.phoneNum = phoneNum;
	}

	public String getFileName() {
		return fileName;
	}

	public void setFileName(String fileName) {
		this.fileName = fileName;
	}

	@Override
	public String toString() {
		return "Candidate [id=" + id + ", name=" + name + ", phoneNum=" + phoneNum + ", fileName=" + fileName + "]";
	}

}

File: src/app/models/candidate.ts


export default class Candidate{
    private id: number;
    private name: string;
    private phoneNum: number;
    private fileName: string;

    constructor(){}

    set setId(id: number){
        this.id = id;
    }

    set setName(name: string){
        this.name = name;
    }

    set setPhNum(phoneNum: number){
        this.phoneNum = phoneNum;
    }

    set setFileName(fileName){
        this.fileName = fileName;
    }

}